html_css/v2/foundations/html-foundations/working-with-text.md

```diff
@@ -0,0 +1,178 @@
+
+Headings are defined much like paragraphs. For example, to create an h1 heading, we wrap our heading text in a `<h1>` tag.
+
+<p class="codepen" data-height="300" data-theme-id="dark" data-default-tab="html,result" data-slug-hash="LYLPLbg" data-user="TheOdinProjectExamples" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;">
+  <span>See the Pen <a href="https://codepen.io/TheOdinProjectExamples/pen/LYLPLbg">
+  html-headings-example</a> by TheOdinProject (<a href="https://codepen.io/TheOdinProjectExamples">@TheOdinProjectExamples</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+<script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
+
+Using the correct levels of headings is important as they provide a hierarchy to the content. An h1 headings should always be used for the heading of the overall page and the lower level headings should be used as the headings for content in smaller sections of the page.
+
+### Strong Element
+
+The `<strong>` element makes text bold. It also semantically marks text as important, this affects things like screen readers that users with visual impairments will rely on to use your website. To define a strong element we wrap text content in a `<strong>` tag.
+
+You can use strong on its own:
+
+<p class="codepen" data-height="300" data-theme-id="dark" data-default-tab="html,result" data-slug-hash="qBjWXrB" data-user="TheOdinProjectExamples" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;">
+  <span>See the Pen <a href="https://codepen.io/TheOdinProjectExamples/pen/qBjWXrB">
+  html-single-strong-example</a> by TheOdinProject (<a href="https://codepen.io/TheOdinProjectExamples">@TheOdinProjectExamples</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+<script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
+
+But you will probably find yourself using the strong element much more in combination with other text elements like this:
+
+<p class="codepen" data-height="300" data-theme-id="dark" data-default-tab="html,result" data-slug-hash="wvewqJr" data-user="TheOdinProjectExamples" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;">
+  <span>See the Pen <a href="https://codepen.io/TheOdinProjectExamples/pen/wvewqJr">
+  html-strong-with-paragraph-exmample</a> by TheOdinProject (<a href="https://codepen.io/TheOdinProjectExamples">@TheOdinProjectExamples</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+<script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
```

Comment:
This also applies to the Em element section afterwards.

So... This might depend on whether the CSS Foundations lesson should include an additional property (to go along with font-weight), but I think it might be worth mentioning _when_ one would want to use the HTML elements vs CSS properties. Like, "if you just want to stylistically make text bold, use a CSS property, but if there is a meaning to text being bold, use the HTMl element" or something. This could def work in either this or the CSS lesson, though, so would love to hear some input on that.

Reply:
I more or less agree.  The wording _is_ correct here to us that know the distinction... 'it semantically marks the text as important'... but that's gonna fly right over the head of first-timers.

I think we should leave this for now and add that distinction in a later lesson. (wherever we talk about font-weight)